boards/realtek/rtl872xda_evb/doc/index.rst

```diff
@@ -0,0 +1,102 @@
+(PSA Level 2), and abundant peripheral resources, it is widely used in smart home appliance,
+line controller, smart door lock, battery camera, smart remote controller, Wi-Fi Speaker, Wi-Fi
+Full MAC NIC, BLE gateway, and smart POS, etc. For more information, check `RTL872XDA-EVB`_.
+
+The features include the following:
+
+- Dual cores: Real-M300 and Real-M200
+- 512KB on-chip SRAM
+- 802.11 a/b/g/n 1 x 1, 2.4GHz + 5GHz
+- Supports BLE 5.0
+- Peripheral Interface:
+
+  - Multi-communication interfaces: SPI x 2, UART x 4, I2C x 2
+  - Hardware Key-Scan interface supports up to 8*8 (64) keys
+  - Hardware IR transceiver can easily adapt to various IR protocols
+  - Supports Real-Time Clock timer together with 10 basic timers
+  - Supports 8 channels of PWM timer and 1 capture timer
+  - Supports 7 channels of general 12-bit ADC and 1 channel of VBAT
+  - Supports 4 channels of touch pad
+  - Supports 8 independent channels of GDMA
+  - Supports USB 2.0 full-speed device mode
+  - Supports SDIO device with 1-bit and 4-bit mode
+  - Embeds a serial LEDC to control the external LED lamps
+  - Integrated Pixel Processing Engine (PPE) to process pixel data faster
+  - Integrated OSPI display interface supports screens with OSPI/QSPI/SPI interfaces
+  - Integrated audio codec supports 2 channels DMIC interface
+  - I2S x 2: up to 384kHz sampling rate
+
+- Cryptographic hardware acceleration (TRNG, ECC, SHA-2, AES)
+
+.. figure:: rtl872xda_evb.webp
+    :align: center
+    :alt: RTL872XDA-EVB
+
+For more information, Get application note and datasheet at `RTL8721Dx Series`_ depending on chip you use.
+
+Supported Features
+==================
+
+Current Zephyr's board target ``rtl872xda_evb`` supports the following features:
+
++------------+------------+-------------------------------------+
+| Interface  | Controller | Driver/Component                    |
++============+============+=====================================+
+| UART       | on-chip    | serial                              |
++------------+------------+-------------------------------------+
+| GPIO       | on-chip    | gpio                                |
++------------+------------+-------------------------------------+
+| PINMUX     | on-chip    | pinctrl                             |
++------------+------------+-------------------------------------+
+| NVIC       | on-chip    | nested vector interrupt controller  |
++------------+------------+-------------------------------------+
+
+Building
+********
+
+Here is an example for building the :zephyr:code-sample:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: rtl872xda_evb
+   :goals: buil
+
+Flashing
+********
+
+When the build finishes, downloading images into flash by `AmebaImageTool`_:
+
+See the ApplicationNote chapter Image Tool from documentation links for more details.
+
+#. Environment Requirements: EX. WinXP, Win 7 or later, Microsoft .NET Framework 4.0.
```

Comment:
so this device is completely unusable on linux or mac? `west flash` also cannot work?

Reply:
We currently do not support the ‘west flash’ command. In the future, we will provide a Python script to enable downloading across all platforms by loguart.